server/trillian_log_signer/main.go

```diff
@@ -144,15 +144,19 @@ func main() {
 	log.QuotaIncreaseFactor = *quotaIncreaseFactor
 	sequencerManager := server.NewSequencerManager(registry, *sequencerGuardWindowFlag)
 	info := server.LogOperationInfo{
-		Registry:            registry,
-		BatchSize:           *batchSizeFlag,
-		NumWorkers:          *numSeqFlag,
-		RunInterval:         *sequencerIntervalFlag,
-		TimeSource:          util.SystemTimeSource{},
-		PreElectionPause:    *preElectionPause,
-		MasterCheckInterval: *masterCheckInterval,
-		MasterHoldInterval:  *masterHoldInterval,
-		ResignOdds:          *resignOdds,
+		Registry:    registry,
+		BatchSize:   *batchSizeFlag,
+		NumWorkers:  *numSeqFlag,
+		RunInterval: *sequencerIntervalFlag,
+		ElectionConfig: election.Config{
+			PreElectionPause:    *preElectionPause,
+			MasterCheckInterval: *masterCheckInterval,
+		},
+		MasterHoldInterval: *masterHoldInterval,
+		// TODO(pavelkalinnikov): ResignOdds initially set a geometric
```

Comment:
Leaving the old flag name for compatibility reasons. Could change it in this PR though.

Reply:
Yes, I'd leave it for now and we can do smaller PRs to migrate it.